Jenkinsfile

```diff
@@ -7,15 +7,8 @@
 import com.sonatype.jenkins.pipeline.GitHub
 import com.sonatype.jenkins.pipeline.OsTools
 
-properties([
```

Comment:
I'm assuming the intent is to replace these with a `run` parameter defined in the UI.

Reply:
> I'm assuming the intent is to replace these with a `run` parameter defined in the UI.

That's correct. Sorry I forgot to mention that :sweat_smile:.